tutorials/configure-dvc-with-object-storage/index.mdx

```diff
@@ -0,0 +1,144 @@
+  - You have [created an API key](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-api-keys/)
+  - You have created a repository to store your metadata.
+  - You have made your [first request with Scaleway API](https://www.scaleway.com/en/developers/api/#quickstart:-first-request)
+  - You [authenticated to the API](https://www.scaleway.com/en/developers/api/#how-to-authenticate) for the first time 
+  - You have installed the [AWS CLI](/storage/object/api-cli/object-storage-aws-cli/)
+</Message> 
+
+## Setting up DVC
+
+1. Run the following command to install the DVC Python package:
+    ```bash
+    pip3 install dvc
+    ```
+
+2. Run the following command to install the S3 dependencies:
+    ```bash
+    pip3 install "dvc[s3]"
+    ```
+
+3. Run the following command in the desired repository to initialize DVC:
+    ```bash
+    dvc init
+    ```
+
+4. Run the following command to commit the initial DVC configuration files:
+    ```bash
+    git commit -m "Initialize DVC"
+    ```
+
+## Retrieving and tracking data
+
+1. Run the following command to copy the example data file to your repository:
+    ```bash
+    dvc get https://github.com/iterative/dataset-registry \
+    get-started/data.xml -o data/data.xml
+    ```
+
+2. Run the following command to stage files you want to store in your bucket:
+    ```bash
+    dvc add data/data.xml
+    ```
+
+3. Run the following command to track the DVC metadata file and `.gitignore` file with git:
+    ```bash
+    git add data/data.xml.dvc data/.gitignore
+    ```
+
+    <Message type="note">
+      The `.gitignore` file contains paths to the data files so they are not tracked by Git.
+    </Message>
+
+4. Run the following command to commit the metadata file:
+    ```bash
+    git commit -m "Add raw data"
+    ```
+
+5. Run the following command to push your changes to your Git repository:
+   ```bash
+   git push
+   ```
+  
+## Pushing data to Scaleway Object Storage
+
+1. Run the following command to add your bucket as a remote storage for your data:
+   ```bash
+   dvc remote add -d  myremote s3://my-bucket/path
+   ```
+
+2. Run the following command to set the S3 endpoint of your remote storage:
+    ```bash
+    dvc remote modify myremote \
+    endpointurl https://s3.fr-par.scw.cloud
+    ```
+
+    <Message type="note">
+      Edit the `endpointurl` according to the geographical region of your bucket. It can either be `fr-par` (Paris, France), `nl-ams` (Amsterdam, The Netherlands) or `pl-waw` (Warsaw, Poland).
+    </Message>
+
+3. Run the following command to push your data to your bucket:
+    ```bash
+    dvc push
+    ```
+
+4. Run the following command to list the content of your bucket:
+    ```bash
+    aws s3api list-objects-v2 --bucket my-bucket
+    ```
+
+    The content of the bucket is displayed:
+
+    ```json
+    {
+      "Contents": [
+          {
+              "Key": "22/a1a2931c8370d3aeedd7183606fd7f",
```

Comment:
It looks like a hash. Are you sure it is the right data? Why is it not named data.xml ?

Reply:
See note below! This is the expected behavior, according to the video tutorial on their doc website: https://youtu.be/kLKBcPonMYw?t=263